src/database/postgresql/postgresql.js

```diff
@@ -0,0 +1,99 @@
+const { Client } = require('pg');
+const exec = require('./../../utils/await-exec');
+const isGzip = require('./../../utils/isGzip');
+const files = require('./../../utils/files');
+const path = require('path');
+const encryptionCheck = require('../../utils/isEncrypted');
+
+let awaitPostgresqlConnect = (connection) => {
+    return new Promise((resolve, reject) => {
+        connection.connect(function (err) {
+            if (err) {
+                reject(err);
+                return;
+            }
+            resolve(connection);
+        });
+    });
+};
+
+let awaitPostgresqlEnd = (connection) => {
+    return new Promise((resolve, reject) => {
+        connection.end(function (err) {
+            if (err) {
+                reject(err);
+                return;
+            }
+            resolve(connection);
+        });
+    });
+};
+
+let connect = async (dbConfig) => {
+    const connection = new Client({
+        user: dbConfig.dbAuthUser,
+        password: dbConfig.dbAuthPwd,
+        host: dbConfig.dbHost,
+        port: dbConfig.dbPort,
+        database: dbConfig.dbName,
+    });
+
+    const connRes = await awaitPostgresqlConnect(connection);
+    const disConnRes = await awaitPostgresqlEnd(connection);
+
+    return connRes;
+};
+
+let dump = async (dbConfig, key, backupPath) => {
+    const postgresqlDumpCmd = `pg_dump --dbname=postgresql://${dbConfig.dbAuthUser}:${dbConfig.dbAuthPwd}@${dbConfig.dbHost}:${dbConfig.dbPort}/${dbConfig.dbName} \
+    --compress=0..9 \
+    --format=c
```

Comment:
Escape the new line

Reply:
Always test the features before committing.

Reply:
Yeah I did. I have checked this file and is not showing me this error.

Reply:
Ok. I fixed it but I will commit after testing the feature

Reply:
Okay, the backup command doesnt work for me (in bash shell) without the escaping the new line.
Try running the synchly commands to test out the features.
For example: Backup a db using the `synchly --run` command to take an instant backup and restore it using `synchly -R`.

Reply:
ok

Reply:
Hey, I tried doing backup and restore. I am getting error in restore but backup is successful.

Reply:
for every table present in database it is showing that the table already exists.

Reply:
![Screenshot from 2020-11-22 15-52-55](https://user-images.githubusercontent.com/46401579/99901201-def4a600-2cda-11eb-8290-4c460d3140e7.png)